K8s-deployment/Charts/databroker/values.yaml

```diff
@@ -0,0 +1,1302 @@
+    existingSecret: "certs"
+    existingSecretFullChain: false
+
+## @param logs Path of the RabbitMQ server's Erlang log file. Value for the `RABBITMQ_LOGS` environment variable
+## ref: https://www.rabbitmq.com/logging.html#log-file-location
+##
+logs: "-"
+## @param ulimitNofiles RabbitMQ Max File Descriptors
+## ref: https://github.com/bitnami/bitnami-docker-rabbitmq#environment-variables
+## ref: https://www.rabbitmq.com/install-debian.html#kernel-resource-limits
+##
+ulimitNofiles: "65536"
+## RabbitMQ maximum available scheduler threads and online scheduler threads. By default it will create a thread per CPU detected, with the following parameters you can tune it manually.
+## ref: https://hamidreza-s.github.io/erlang/scheduling/real-time/preemptive/migration/2016/02/09/erlang-scheduler-details.html#scheduler-threads
+## ref: https://github.com/bitnami/charts/issues/2189
+## @param maxAvailableSchedulers RabbitMQ maximum available scheduler threads
+## @param onlineSchedulers RabbitMQ online scheduler threads
+##
+maxAvailableSchedulers: ""
+onlineSchedulers: ""
+
+## The memory threshold under which RabbitMQ will stop reading from client network sockets, in order to avoid being killed by the OS
+## ref: https://www.rabbitmq.com/alarms.html
+## ref: https://www.rabbitmq.com/memory.html#threshold
+##
+memoryHighWatermark:
+  ## @param memoryHighWatermark.enabled Enable configuring Memory high watermark on RabbitMQ
+  ##
+  enabled: true
+  ## @param memoryHighWatermark.type Memory high watermark type. Either `absolute` or `relative`
+  ##
+  type: "relative"
+  ## Memory high watermark value.
+  ## @param memoryHighWatermark.value Memory high watermark value
+  ## The default value of 0.4 stands for 40% of available RAM
+  ## Note: the memory relative limit is applied to the resource.limits.memory to calculate the memory threshold
+  ## You can also use an absolute value, e.g.: 256MB
+  ##
+  value: 0.4
+
+## @param plugins List of default plugins to enable (should only be altered to remove defaults; for additional plugins use `extraPlugins`)
+##
+plugins: "rabbitmq_management rabbitmq_peer_discovery_k8s rabbitmq_prometheus"
+## @param communityPlugins List of Community plugins (URLs) to be downloaded during container initialization
+## Combine it with extraPlugins to also enable them.
+##
+communityPlugins: ""
+## @param extraPlugins Extra plugins to enable (single string containing a space-separated list)
+## Use this instead of `plugins` to add new plugins
+##
+extraPlugins: "rabbitmq_auth_backend_ldap"
+
+## Clustering settings
+##
+clustering:
+  ## @param clustering.enabled Enable RabbitMQ clustering
+  ##
+  enabled: true
+  ## @param clustering.addressType Switch clustering mode. Either `ip` or `hostname`
+  ##
+  addressType: hostname
+  ## @param clustering.rebalance Rebalance master for queues in cluster when new replica is created
+  ## ref: https://www.rabbitmq.com/rabbitmq-queues.8.html#rebalance
+  ##
+  rebalance: false
```

Comment:
make this true. want the queues master to be rebalanced on addition of replicas

Reply:
Done